### DISCUSSION TITLE
Notification to Webex via Webhook

### DISCUSSION THREAD

**Discussion by MacGovery** [hashtopolis/server#1048] [Community Q&A]
Hello,

i'm struggling to set-up a Notification to a Webex Channel via a WebHook.
My command looks like:
`curl -X POST -H "Content-Type: application/json" -d '{"text" : "This is a message from a Webex incoming webhook."}' "https://webexapis.com/v1/webhooks/incoming/secret"`
I tried it as ChatBot and Discord Webhook. Both do not work.

What am I doing wrong?

**Comment by zyronix** [comment]
I think you need to do quite a lot more for a webex notification.

https://developer.webex.com/docs/api/v1/messages/create-a-message

And the secret needs to be a bearer token:
```
curl --request POST  --header "Authorization: Bearer ACCESS_TOKEN" ...
```

See https://developer.webex.com/docs/basics
